src/components/Profile.js

```diff
@@ -0,0 +1,33 @@
+import { useSelector } from 'react-redux';
+import DisplayMissions from './DisplayMissions';
+
+function Profile() {
+  const rocketData = useSelector((state) => state.rockets.Data);
+  const bookedRockets = rocketData.filter((rockets) => rockets.reserved === true);
+  return (
+    <div className="profile">
+      <div className="missions">
+        <DisplayMissions />
+      </div>
+      <div className="rockets">
+        <h2 className="rocketsHeader">Rockets</h2>
+        {bookedRockets.length === 0 && (
+          <p>No rockets have been reserved yet!</p>
```

Comment:
- Kudos for adding a placeholder message here! 👏

Reply:
@tugsem, Thank you for the great review 👏. We will get right on it and fix⚙️ these issues.